docs/how-to-run.md

```diff
@@ -0,0 +1,32 @@
+# How to run a karton service?
+
+<img src="../img/logo.svg" width="64">
+
+This is a short tutorial for people who stumbled upon a Karton service and want to know how to use it.
+
+### The karton way
+
+Karton is a modular framework dedicated for malware analysts. Karton services
+are not usually run independently. Think of them as "plugins", elements of a
+bigger pipeline.
+
+A simple Karton deploymet, plus mwdb as a backend, looks like this:
+
+<img src="../img/playground.png" width="600">
+
+The question marks is where other Karton services go.
```

Comment:
To me this sentence sounds a little weird. How about:

> Question marks are placeholders for other Karton services.

Reply:
the goal of that sentence was "your services will take place of the question marks", but I like the word "placeholder" :thinking: